src/chrome/content/rules/Digital_News_Agency.com.xml

```diff
@@ -1,41 +1,11 @@
-<!--
-	CDN buckets:
+<ruleset name="Digital News Agency.com">
 
-		- dna-bucket-dna.cf.rawcdn.com <=> df7r7v78mcbtc.cloudfront.net
-
-
-	^digitalnewsagency.com: Cert only matches www
-
-
-	Mixed content:
-
-		- Images from dna-bucket-dna.cf.rawcdn.com *
-
-	* Secured by us
-
--->
-<ruleset name="Digital News Agency.com" default_off="expired">
-
-	<!--	Direct rewrites:
-				-->
-	<target host="www.digitalnewsagency.com" />
-
-	<!--	Complications:
-				-->
 	<target host="digitalnewsagency.com" />
+	<target host="www.digitalnewsagency.com" />
 	<target host="dna-bucket-dna.cf.rawcdn.com" />
 
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^\.digitalnewsagency\.com$" name="^_dna_session$" /-->
-
 	<securecookie host="^\.digitalnewsagency\.com$" name=".+" />
 
-
-	<rule from="^http://digitalnewsagency\.com/"
-		to="https://www.digitalnewsagency.com/" />
-
 	<rule from="^http://dna-bucket-dna\.cf\.rawcdn\.com/"
 		to="https://df7r7v78mcbtc.cloudfront.net/" />
```

Comment:
That looks a bit dangerous. Can you provide a test url for it? Otherwise you should remove this target.

Reply:
@J0WI ```
> nslookup
Default Server:  UnKnown
Address:  192.168.0.1

> dna-bucket-dna.cf.rawcdn.com
Server:  UnKnown
Address:  192.168.0.1

Non-authoritative answer:
Name:    df7r7v78mcbtc.cloudfront.net
Addresses:  13.33.23.17
          13.33.23.146
          13.33.23.85
          13.33.23.82
          13.33.23.240
          13.33.23.121
          13.33.23.71
          13.33.23.35
Aliases:  dna-bucket-dna.cf.rawcdn.com
```

Reply:
@J0WI yeah, I'd actually initially removed it, but I added it back because I realised it's valid and it's a little beyond the scope of this PR to mess with individual rulesets too much. I've added a test URL, and I don't think there are any issues with this rewrite (possibly it's for third-parties, as I haven't seen any mixedcontent issues causing a need for the rewrite on the main site).